### PR TITLE
feat: add Neutrino to Litecoin LND

### PR DESCRIPTION
**Pull request by michael1011**
This PR:

- updates LND to version `0.9.2-beta`. Our simulation tests are updated already: https://github.com/ExchangeUnion/xud/pull/1436
- allows LND to sync with Neutrino on the Litecoin chain. This is possible by using a [forked version of LND](https://github.com/ltcsuite/lnd) and the fact that @kilrau and me are running public LTCD nodes to serve the Neutrino filters. Feel free to join the cause and add additional Neutrino seed nodes in case you are running a public BTCD or LTCD node on testnet or mainnet

The LND fork for Litecoin is still on `0.9.0-beta` but that shouldn't be a problem for xud

### CODE REVIEW

**Comment by erkarl** [comment]
Could you please also upgrade the sample config files for ltc to contain `neutrino` option?

File: images/utils/launcher/config/config.py [hunk 1]
```diff
@@ -503,6 +504,10 @@ def apply_network_args(self):
             if "bitcoind" in self.nodes:
                 self.nodes["bitcoind"]["mode"] = "neutrino"
 
+        if hasattr(self.args, "litecoin_neutrino"):
+            if "bitcoind" in self.nodes:
```
Comment (reliveyy) [inline images/utils/launcher/config/config.py]: Should be "litecoind" here.

**Comment by erkarl** [comment]
In general seems to work nicely - got synced in less than ~30 mins.

File: images/utils/launcher/config/config.py (hunk 1 above)
Comment (michael1011) [reply images/utils/launcher/config/config.py]: Thanks for catching that!

**Pull request merged**